types/node/globals.d.ts

```diff
@@ -13,6 +13,13 @@ interface ErrorConstructor {
     stackTraceLimit: number;
 }
 
+interface Error {
+  /**
+   * @see https://nodejs.org/dist/latest/docs/api/errors.html#nodejs-error-codes
+   */
+  code?: string | number;
```

Comment:
`code` is documented as only string here, can it really be a `number`?

ref: https://nodejs.org/dist/latest/docs/api/errors.html#errorcode_1

Reply:
Yes. The subprocess module set a numeric exit status here:

```js
import {it} from 'node:test';
import {rejects} from 'node:assert/strict';
import {stat} from 'node:fs/promises';
import {exec} from 'node:child_process';
import {promisify} from 'node:util';

it('Error with string code', async() => rejects(
  async() => stat('unexistsFile'), 
  e => e instanceof Error && typeof e.code === 'string'),
);

const asyncExec = promisify(exec);

it('Error with number code', async() => rejects(
  async() => asyncExec('unexistsCommand'),
  e => e instanceof Error && typeof e.code === 'number'),
);
```